sample-app/src/components/InputDropdown.tsx

```diff
@@ -8,14 +8,17 @@ interface Props {
   placeholder?: string,
   screenReaderInstructions: string,
   screenReaderInstructionsId: string,
-  options: Option[],
+  options: {results: Option[], label?: string}[],
```

Comment:
This changes forcers consumers of this component to organize the dropdown into sections. Is there a way we could avoid that?

Reply:
The only way I could think of to avoid that would be to keep the options array as is, but pass in an optional array of strings and numbers for the labels of each section and the number of results in each section. This would assume that the options array is ordered by section in the same order as the labels and counts.

Something like `additionalData?: {labels: string[], counts: number[]}` or 
`additionalData?: {label: string, count: number}[]`

Reply:
another way would be allowing for both sections and the previous array of options 
`options: {results: Option[], label?: string}[] | Option[]`
and either InputDropdown support both logic, or transform options to a section internally if it's not one

Reply:
How about keeping the options data the same but making it optional, and adding an optional sections param with a different interface?

Reply:
would it seems a little odd if there's two params for passing options?